### ISSUE TITLE
Children of abnormal tumor vascularization

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3171]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)
1) abnormal tumor perivascular space
2) increased tumor perivascular space
3) decreased tumor perivascular space

>2. Synonyms (e.g., absent spleen)
1) abnormal tumor Virchow-Robin space
2) increased tumor Virchow-Robin space
3) decreased tumor Virchow-Robin space

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).
1) An anomaly in the extracellular space bounded by capillary endothelium on one side and parenchymal basement membrane on the other side.
2) An enlargement of the extracellular space bounded by capillary endothelium on one side and parenchymal basement membrane on the other side.
3) A reduction in the extracellular space bounded by capillary endothelium on one side and parenchymal basement membrane on the other side.
PMID:16416090

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)
1) abnormal tumor vascularization (MP:0010144)
2) abnormal tumor perivascular space (new term)
3) abnormal tumor perivascular space (new term)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)
RGD:cp
